runtime/plugin/editorconfig.lua

```diff
@@ -1,11 +1,13 @@
-if vim.g.editorconfig_enable == false or vim.g.editorconfig_enable == 0 then
-  return
-end
```

Comment:
probably still want this--users who want to fully opt-out won't want the autocmd to be created

Reply:
We would need to create a separate variable then, as otherwise setting `editorconfig_enable` after the fact would not re-create the autocommand.

The autocommand is basically a no-op with `editorconfig_enable` set to false, but if someone doesn't even want the autocommand listed at all they can always `au! editorconfig`.